src/extension.ts

```diff
@@ -62,14 +64,24 @@ interface LogChannel {
   hasMoreLogs?: boolean;
 }
 
-export function activate(context: vscode.ExtensionContext) {
+export const loadEnvironments = async (
+  environmentsDataProvider: Env0EnvironmentsProvider,
+  environmentsTree: vscode.TreeView<Environment>
+) => {
+  environmentsTree.message = `loading environments...`;
+  const currentBranch = await getCurrentBranchWithRetry();
+  environmentsTree.message = `loading environments from branch ${currentBranch}...`;
```

Comment:
I don't like the message, but I may be nitpicking. @liranfarage89 - what's your take?
I would prefer 
> `loading environments targeting ${current-branch}...`